.devcontainer/Dockerfile

```diff
@@ -8,16 +8,74 @@ FROM mcr.microsoft.com/vscode/devcontainers/go:0-${VARIANT}
 ARG NODE_VERSION="none"
 RUN if [ "${NODE_VERSION}" != "none" ]; then su vscode -c "umask 0002 && . /usr/local/share/nvm/nvm.sh && nvm install ${NODE_VERSION} 2>&1"; fi
 
-# Download the Microsoft repository GPG keys
-RUN wget https://packages.microsoft.com/config/debian/10/packages-microsoft-prod.deb
-# Register the Microsoft repository GPG keys
-RUN  dpkg -i packages-microsoft-prod.deb
+# Download and register the Microsoft repository GPG keys
+RUN wget https://packages.microsoft.com/config/debian/10/packages-microsoft-prod.deb && \
+    dpkg -i packages-microsoft-prod.deb && \
+    rm -f packages-microsoft-prod.deb
 
 # [Optional] Uncomment this section to install additional OS packages.
 RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
-    && apt-get install -y --no-install-recommends fish\
-    && apt-get install -y --no-install-recommends powershell
+    && apt-get install -y --no-install-recommends \
+        fish \
+        powershell \
+        tmux \
+    && apt-get clean
 
+# Deploy latest stable oh-my-posh as a starting point:
+RUN pwsh -Command Install-Module oh-my-posh -Scope AllUsers -Force && \
```

Comment:
We'd need to add a compilation step because that folder doesn't have any of the binaries.

Alternatively I could download the good binary via wget and plop it in the right place; it just seemed simpler this way.

If you wanted technically we could compile OMP at build time, but I opted against it because I figured it was safer to grab the latest "stable" released version instead of the freshest main branch.